wordle-web/components/NameDialog.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <v-dialog v-model="modelValue" max-width="500">
+    <v-card>
+      <v-sheet color="primary"></v-sheet>
+      <v-card-title>Enter a username.</v-card-title>
+      <v-text-field v-model="userName" label="UserName" persistent></v-text-field>
+      <v-card-actions>
+        <v-spacer></v-spacer>
+        <v-btn color="primary" @click="closeDialog">Cancel</v-btn>
+        <v-btn color="primary" @click=saveUsername>Save</v-btn>
+      </v-card-actions>
```

Comment:
Dialogue needs to be persisted. Here is the documentation for this attribute, you can look at the source code to see how it's accomplished

https://vuetifyjs.com/en/components/dialogs/#persistent